kclvm/sema/src/resolver/arg.rs

```diff
@@ -81,11 +86,15 @@ impl<'ctx> Resolver<'ctx> {
                 Some(param) => param.ty.clone(),
                 None => {
                     if !func_ty.is_variadic {
+                        let expect_err_msg = if func_ty.params.len() > 1 {
```

Comment:
I suggest maintaining a separate function or macro function in the compiler base error module.

Reply:
fixed